.goreleaser.yaml

```diff
@@ -9,15 +9,22 @@ builds:
       - CGO_ENABLED=0
     goos:
       - linux
-      - windows
```

Comment:
nit:

Is this change (and the ones in `archives.replacements`) technical necessary, or systems favorism 😋?
If not technical necessary I would like to opt for alphabetical order